cmd/minikube/cmd/start.go

```diff
@@ -156,6 +171,56 @@ func runStart(cmd *cobra.Command, args []string) {
 	clusterUp(&config)
 }
 
+// Set Docker Proxy
+func setDockerProxy () {
```

Comment:
+1,  I think this is better and easier to understand, even though you have to repeat the if statements